scene/gui/text_edit.cpp

```diff
@@ -1418,11 +1417,49 @@ void TextEdit::_notification(int p_what) {
 					int l = line_from + i;
 					ERR_CONTINUE(l < 0 || l >= completion_options.size());
 					Color text_color = cache.completion_font_color;
+
+					if (syntax_coloring) {
+						String upper = completion_options[l].to_upper();
+
+						if (keywords.has(completion_options[l])) {
+							if (completion_options[l].get(0) == L'_' || completion_options[l].get(0) != upper[0]) {
+								// colorize functions name
+								text_color = cache.function_color;
+							} else {
+
+								if (completion_options[l] == upper &&
+										completion_options[l] != "IP" &&
+										completion_options[l] != "RID" &&
+										completion_options[l] != "AABB" &&
+										completion_options[l] != "JSON" &&
+										completion_options[l] != "OS")
+									// colorize constants
+									text_color = cache.keyword_color;
+								else
+									// colorize engine class names
+									text_color = cache.basetype_color;
+							}
+						} else if (completion_options[l] == upper) {
+							// colorize constants
+							text_color = cache.keyword_color;
```

Comment:
also constants are not keywords, so if we want meaningful colors, we should use simply left this as text color, or maybe use the member color.

Reply:
I thinked, that it would be better.. ok I comment this line